source/transactions/tests/retryable-commit-errorLabels.yml

```diff
@@ -0,0 +1,261 @@
+runOn:
+    -
+        minServerVersion: "4.3.0"
+        topology: ["replicaset", "sharded"]
+
+database_name: &database_name "transaction-tests"
+collection_name: &collection_name "test"
+
+data: []
+
+tests:
+  - description: commitTransaction fails with RetryableWriteError label after two network exceptions
```

Comment:
This test might be redundant in light of "commitTransaction fails after two errors". Perhaps we can add a "RetryableWriteError" assertion to that test instead.